content/en/guide/v10/differences-to-react.md

```diff
@@ -85,6 +85,26 @@ standard `class` attribute instead of `className`.
 
 Most Preact developers prefer to use `class` because it's shorter to write, but both are supported.
 
+### Raw SVG inside JSX
+
+SVG is pretty interesting when it comes to the names of its properties and attributes. Some properties (and their attributes) on SVG objects are camelCased (e.g. [clipPathUnits on a clipPath element](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/clipPath#Attributes)), some attributes are kebab-case (e.g. [clip-path on many SVG elements](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/Presentation)), and other attributes (usually ones inherited from the DOM, e.g. `oninput`) are all lowercase.
+
+Preact doesn't normalize SVG properties and attributes [like React does]((https://github.com/facebook/react/pull/6243)) and instead, allows you to write valid SVG inside JSX by passing attributes through.
```

Comment:
What are your thoughts about rephrasing that section to show the advantages of Preact's approach? Something akin to:

```
Preact forwards SVG-Attributes as is. This allows you to copy and paste unmodified SVG snippets right into your code and have them work out of the box. This allows greater interoperability with tools designers tend to use to generate icons or SVG illustrations.

If you're coming from React you're likely used to specify every attribute in camelCase. If you'd like to continue using the camelCase'd attribute names you can use our [preact/compat] compatibility layer. It mirrors the React API and normalizes these attributes.
```

What do you think?

Reply:
Yes! My initial intent was to phrase it as a feature more than un-react thing. Should I drop the first paragraph about SVG? I realize it's a bit broad and out of scope. I copied it verbatim since it was very useful when I was investigating this particular issue.

Reply:
Noo, I love the first paragraph here! It makes the doc sound way more "human" 🙌 I'd just change the second one as it can be interpreted wrongly when potential users read "Preact doesn't normalize SVG properties and attributes" without there being an explanation why we made that choice.

Reply:
done!